heroic-component/src/main/java/com/spotify/heroic/metric/QueryMetricsResponse.java

```diff
@@ -84,6 +88,8 @@ public void serialize(
             g.writeFieldName("result");
             serializeResult(g, common, result);
 
+            g.writeObjectField("rawSampleSize", response.getPreAggregationSampleSize());
```

Comment:
Field should be named the same as the resulting JSON object to reduce confusion (if possible).